e2eTests/.gitignore

```diff
@@ -55,6 +55,11 @@ fastlane/screenshots
 **/.fructose/components.js
 **/.fructose/components.test.js
 
+!dist
```

Comment:
the whole of dist should probably be ignored and you can check in assets separately